src/main/schemas/territories.schema.json

```
@@ -15,7 +15,7 @@
       },
       "dcncTerritory": {
         "type": "string",
-        "pattern": "^[a-zA-Z ']+$"
+        "pattern": "^[a-zA-ZÀ-ÿ ']+$"
```

What about allowing all Unicode characters but for control characters?
```suggestion
        "pattern": "^[^\x00-\x1f\x7f\x80-\x97]+$"
```

---

Updated via [43d5ff3](https://github.com/ISDCF/registries/pull/440/commits/43d5ff3da268f5191b787abd4c0fe93623899c64)